ui/app/AppLayouts/Chat/panels/communities/CommunityChannelsAndCategoriesBannerPanel.qml

```diff
@@ -23,7 +23,8 @@ Rectangle {
     radius: 16
     color: Style.current.transparent
     property string communityId
-
+    signal addMembersClicked()
```

Comment:
Are we declaring signals after object [properties](https://doc.qt.io/qt-5/qml-codingconventions.html#qml-object-declarations)

Reply:
@MishkaRogachev it's declared after a declared property which was already there, do you mean that I should move the existing property together my new declared signal above the object's properties?

Reply:
Yep, i thought we've discussed it here: [https://github.com/status-im/StatusQ/pull/594](https://github.com/status-im/StatusQ/pull/594)